Binance.Net/BinanceHelpers.cs

```diff
@@ -20,8 +20,24 @@ public static class BinanceHelpers
             if (headers == null)
                 return null;
 
-            var headerValues = headers.SingleOrDefault(s => s.Key == "X-MBX-USED-WEIGHT").Value;
-            if (int.TryParse(headerValues.First(), out var value))
+            var headerValues = headers.SingleOrDefault(s => s.Key.StartsWith("X-MBX-USED-WEIGHT-")).Value;
```

Comment:
why is this changed? The X-MBX-USED-WEIGHT is still returned right?

Reply:
On futures, it looks like this` X-MBX-USED-WEIGHT-(intervalNum)(intervalLetter)`, for combining the spot and futures, I think it will look like this.
In the future, will need to create BinanceFuturesHelper. cs

Reply:
By the way in the spot exactly the same thing is done
https://binance-docs.github.io/apidocs/spot/en/#limits